src/components/Controls.vue

```diff
@@ -11,23 +25,27 @@
 </template>
 
 <script lang="ts">
-import Vue from 'vue'
-import { mapActions, mapState } from 'vuex'
+import Vue from "vue";
```

Comment:
@navyhockey56 see `tslint.json` for rules.

There is a rule `"quotemark": [true, "single"]` - you'll want to change that to `"quotemark": [true, "double"]` if you prefer double quotes.